share/completion/cmus-remote

```diff
@@ -0,0 +1,42 @@
+# (C) 2025 unrealapex
+
+# Completion script for cmus-remote
+
+function completion/cmus-remote {
+
+        typeset OPTIONS ARGOPT PREFIX
+        OPTIONS=( #>#
+        "--server; connect using socket SOCKET instead of \$XDG_RUNTIME_DIR/cmus-socket"
+        "--passwd; password to use for tcp/ip connection"
```

Comment:
Is this format okay for long flags that take a value without using `=`?

Reply:
The word following the `--passwd` option would not be completed as a password, but that would not matter since there are no password completion settings in this script now.